app/src/main/java/de/hhn/aib/labsw/blackmirror/view/widgets/AbstractWidget.java

```diff
@@ -30,6 +30,40 @@ public void setPosition(Position pos) {
                     screen.height / 2 - this.getHeight() / 2);
         }
     }
-}
 
+    public void setPosition(int xPosition, int yPosition) {
+        Dimension screen = Toolkit.getDefaultToolkit().getScreenSize();
+        Dimension pos = new Dimension(xPosition, yPosition);
```

Comment:
Use java.awt.Point instead of Dimension as this describes a position not an area

Reply:
Valid suggestions, however I would rather not touch this part any more, since the implementation was just a little workaround until this whole class has to be changed with an upcoming user story (size fitting & horizontal layout etc.).

I also just realised that there is currently a bug on side of the app, where the app sends(creates) incorrect x/y coordinates for the 3th column.